[BUG/DEPRECATION] Remove fused attention/mlp

Qubitium opened the pull request:
Reason for PR:

* Broken since at least March 1st with no fix in sight
* No longer have performance value compared to marlin and new marlin kernels (vllm)
* Why fix/maintain something that no one would want to use in production env?

Ref: Issue #655 

Changes:

* Remove all fused attention/mlp code

TESTS
* [x] PASS: test_quantization
* [x] PASS: test_serializtion
* [x] PASS: test_triton
* [x] PASS: test_q4

fxmarty commented:
Hey @Qubitium sorry for taking time to review. I've had too many other things in my plate, I'll try to review everything by Friday. Do you have an email/slack/twitter I could reach out to you at? I feel like for some non-huge PRs/bugfixes it would be simplier if you had write access to the main branch. What do you think?

Qubitium commented:
@fxmarty  Sure thing! You can reach me via twitter @ https://twitter.com/qubitium or email at qubitium@lbx.dev.  Willing to help out anyway I can with the project. Most importantly, I am willing to put in the grunt work to get all the tensor-math (cuda/triton) math level things working, fixed, and added.
